srcpkgs/acpi_call-dkms/template

```diff
@@ -1,23 +1,23 @@
 # Template file for 'acpi_call-dkms'
 pkgname=acpi_call-dkms
-version=1.2.0
-revision=3
+version=1.2.1
+revision=1
 archs="i686 x86_64*"
 wrksrc=acpi_call-${version}
 short_desc="Kernel module allowing calls to ACPI methods through /proc/acpi/call"
 maintainer="Daniel A. Maierhofer <git@damadmai.at>"
 license="GPL-3.0-or-later"
-homepage="https://github.com/damadmai/acpi_call"
-distfiles="$homepage/archive/v$version.tar.gz"
-checksum=c48a0a8768d657a8da7b59d7dbf6276554313fa6a0cb012fa4cf85a77e55f44b
+homepage="https://github.com/nix-community/acpi_call"
+distfiles="${homepage}/archive/refs/tags/v${version}.tar.gz"
+checksum=aa84ac8a209e18ee6de5fa809abd3474fdca67cda3c3256f0650c4fce78d4a0f
 
 dkms_modules="acpi_call ${version}"
 depends="dkms"
 
 do_install() {
 	vmkdir usr/src/acpi_call-${version}
 	vcopy Makefile usr/src/acpi_call-${version}
-	vcopy dkms.conf usr/src/acpi_call-${version}
+	vcopy dkms.conf.in usr/src/acpi_call-${version}/dkms.conf
```

Comment:
In `do_build` you should call `make dkms-conf`, so the `dkms.conf` file with a filled version field is generated.

Reply:
Sure, but I need a bit more help :)
```
=> acpi_call-dkms-1.2.1_1: running do_build ...
make: *** No rule to make target 'dkms-conf'.  Stop.
=> ERROR: acpi_call-dkms-1.2.1_1: do_build: 'make dkms-conf' exited with 2
=> ERROR:   in do_build() at srcpkgs/acpi_call-dkms/template:18
```

Reply:
Typo! `make dkms.conf` is what I meant.

Reply:
Thanks.